src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 1px 1px;
+
+  width: $columns-quantity * $square-size + 20px;
```

Comment:
What is 20px? Would be great to use variables to avoid 'magic numbers'

Reply:
Oops, I could have used existing variables there...